README.md

```diff
@@ -276,11 +331,23 @@ If you already have a pre-commit executable file and you want to use gg-shield,
 all you need to do is to add this line in the file:
 
 ```shell
-ggshield scan --mode pre-commit
+ggshield scan pre-commit
 ```
 
 Do not forget to add your [GitGuardian API Key](https://dashboard.gitguardian.com/api/v1/auth/user/github_login/authorize?utm_source=github&utm_medium=gg_shield&utm_campaign=shield1) to the `GITGUARDIAN_API_KEY` environment variable of your project or development environment.
 
+# Pre-receive
```

Comment:
```suggestion
# Git pre-receive hooks
```

Reply:
Fixed in following commit, index also required change